include/Clusterer.h

```diff
@@ -11,8 +13,24 @@ class Clusterer
   Clusterer() = default;
   ~Clusterer() = default;
 
- private:
+  // Use this if you already know which clusters/signals you want to have
+  inline void setClusters (std::vector<Cluster> &clusterVec) { mClusterVec = clusterVec; }
+
+  // If it is not clear which signals should be taken into your clusters, use this
+  // This function loops through the signals in the given CTS Event,
+  // applies some cuts: allowed distance cut in position (fibers) and time (Work In Progress; Not Yet Implemented) 
+  // And gives back the vector of found clusters.
```

Comment:
FYI: You could use `///` instead of `//` (doxygen) and make the last comment line before the actual code `@todo Implement time cut` or similar to add to the doxygen todo list.